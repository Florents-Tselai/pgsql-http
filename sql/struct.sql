CREATE EXTENSION http;

select http_struct(('GET', '"example.com"', NULL, NULL, NULL)::http_request);