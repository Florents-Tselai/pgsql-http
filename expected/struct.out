CREATE EXTENSION http;
-- select http_struct(('GET', 'https://example.com', NULL, 'text/json', 'content')::http_request);
SELECT status, content_type FROM http_struct_get('https://httpbin.org/status/202');
 status |       content_type       
--------+--------------------------
    202 | text/html; charset=utf-8
(1 row)

